test/validation/org/apache/cassandra/bridges/ccmbridge/CaptureNodeLogs.java

```diff
@@ -0,0 +1,126 @@
+/*
+ *      Copyright (C) 2012-2014 DataStax Inc.
```

Comment:
Okay, that file has it's own license header because we copied that code from DataStax java driver. For literally every other class, use the license header found in literally ever other source code file. You just ran into bad luck here, nbd.